_text.scss

```diff
@@ -1,176 +1,370 @@
+  }
+  .t-left--screen-xl {
+    text-align: left;
+  }
+  .t-right--screen-xl {
+    text-align: right;
+  }
+}
 
-/* responsive text.  no changes by default, this is all up to you.  */
-@media only screen and (min-width: $media--l) { 
-	.h-xxl--screen-l, 
-	.t-xxxxl--screen-l,
-	.t-triple--screen-l	{ font-size: 3em; }
-    .h-xl--screen-l, 
-	.t-xxxl--screen-l,
-	.t-double--screen-l	{ font-size: 2em; }
-	.h-l--screen-l, 
-	.t-xxl--screen-l		{ font-size: 1.75em; }
-    .h-m--screen-l, 
-	.t-xl--screen-l,
-	.t-xl--screen-l		{ font-size: 1.5em; }
-	.h-s--screen-l, 
-	.t-l--screen-l			{ font-size: 1.33em;}
-	.h-xs--screen-l, 
-	.t-m--screen-l			{ font-size: 1.1667em; }
-	.t-s--screen-l			{ font-size: 0.9167em; }
-	.t-xs--screen-l			{ font-size: 0.8333em; }
-    .t-nat--screen-l 		{ font-size: 1rem; }
-    .t-half--screen-l 		{ font-size: 0.5em; }
-	.t-n--screen-l			{ font-size: 1em; }
-    
-    .t-mid--screen-l, .t-center--screen-l { text-align: center; }
-    .t-left--screen-l{ text-align: left; }
-    .t-right--screen-l { text-align: right; }
-}
-@media only screen and (min-width: $media--xl) { 
-	.h-xxl--screen-xl, 
-	.t-xxxxl--screen-xl,
-	.t-triple--screen-xl	{ font-size: 3em; }
-    .h-xl--screen-xl, 
-	.t-xxxl--screen-xl,
-	.t-double--screen-xl	{ font-size: 2em; }
-	.h-l--screen-xl, 
-	.t-xxl--screen-xl		{ font-size: 1.75em; }
-    .h-m--screen-xl, 
-	.t-xl--screen-xl,
-	.t-xl--screen-xl		{ font-size: 1.5em; }
-	.h-s--screen-xl, 
-	.t-l--screen-xl		{ font-size: 1.33em;}
-	.h-xs--screen-xl, 
-	.t-m--screen-xl		{ font-size: 1.1667em; }
-	.t-s--screen-xl			{ font-size: 0.9167em; }
-	.t-xs--screen-xl		{ font-size: 0.8333em; }
-    .t-nat--screen-xl 		{ font-size: 1rem; }
-    .t-half--screen-xl 	{ font-size: 0.5em; }
-	.t-n--screen-xl			{ font-size: 1em; }
-    
-    .t-mid--screen-xl, .t-center--screen-xl { text-align: center; }
-    .t-left--screen-xl { text-align: left; }
-    .t-right--screen-xl { text-align: right; }
-}
-@media only screen and (max-width: $media--m) { 
-    .h-xxl--screen-m, 
-	.t-xxxxl--screen-m,
-	.t-triple--screen-m	{ font-size: 3em; }
-    .h-xl--screen-m, 
-	.t-xxxl--screen-m,
-	.t-double--screen-m	{ font-size: 2em; }
-	.h-l--screen-m, 
-	.t-xxl--screen-m		{ font-size: 1.75em; }
-    .h-m--screen-m, 
-	.t-xl--screen-m,
-	.t-xl--screen-m		{ font-size: 1.5em; }
-	.h-s--screen-m, 
-	.t-l--screen-m			{ font-size: 1.33em;}
-	.h-xs--screen-m, 
-	.t-m--screen-m			{ font-size: 1.1667em; }
-	.t-s--screen-m			{ font-size: 0.9167em; }
-	.t-xs--screen-m			{ font-size: 0.8333em; }
-    .t-nat--screen-m 		{ font-size: 1rem; }
-    .t-half--screen-m 		{ font-size: 0.5em; }
-	.t-n--screen-m			{ font-size: 1em; }
-    
-    .t-mid--screen-m, .t-center--screen-m { text-align: center; }
-    .t-left--screen-m { text-align: left; }
-    .t-right--screen-m { text-align: right; }
+@media only screen and (max-width: $media--m) {
+  .h-xxl--screen-m,
+  .t-xxxxl--screen-m,
+  .t-triple--screen-m {
+    font-size: 3em;
```

Comment:
not your fault, but I just noticed in going through that we moved all of these sizes to vars, but not in the media queries.  oops.  we should probably update that!